templates/tweets/home.html

```diff
@@ -4,4 +4,15 @@
 
 {% block content %}
 <h1>Homeです</h1>
+{% for tweet in tweets %}
+<hr>
+<ul>
+  <li>{{ tweet.content }}</li>
+  <li><a href="{% url 'accounts:user_profile' tweet.user.username %}">{{tweet.user.username }}</a></li>
+  <li>{{ tweet.created_at }}</li>
+  <li>{{ tweet.id }}</li>
+  <li><a href="{% url 'tweets:detail' tweet.pk %}">詳細</a></li>
```

Comment:
```suggestion
  <li><a href="{% url 'tweets:detail' tweet.id %}">詳細</a></li>
```
[nits] こちらのほうが個人的には好みです。ツイートのIDを取得するということが明示できます。今回の場合、ほかが`pk`で統一されているのでここを直す必要はないかと思います。